maven/maven.bzl

```diff
@@ -267,8 +267,7 @@ def _unsupported_keys(keys_list):
 
 def _fix_string_booleans(value):
     if type(value) == type(""):
-        if value != "True" or value != "true":
-            return False
+        return value == "True" or value == "true"
```

Comment:
`return value.lower() == "true"`?

Reply:
🤦‍♂️ 

... done.

Reply:
So…`TrUe` isn't true? I don't even want to know, do I?

Reply:
No, it is - this is me just being slightly more tolerant, because the starlark type error is confusing.